### PR TITLE
Split storybook urls

### PR DESCRIPTION
**Pull request by mark-tate**

### CODE REVIEW

**Comment by mark-tate** [comment]
This created another iteration on naming and alignment across Sources.

Storybook url became 'storiesUrl' and 'storyUrlPrefix'
It generates Pages that have a `link` and a 'contentUrl' meta.
Used that naming for the Figma pages as well, so 'thumbnailUrl' became 'contentUrl'

File: packages/source-storybook/src/index.ts [hunk 1]
```diff
@@ -23,7 +23,8 @@ export const schema = baseSchema.merge(
       .array(
         z.object({
           description: z.string(),
-          url: z.string(),
+          storiesUrl: z.string().url(),
+          storyUrlPrefix: z.string().url(),
```
Comment (DavieReid) [inline packages/source-storybook/src/index.ts:27]: Should this be optional?  If not provided `storiesUrl` will be used.

**Review by tambo — APPROVED** [review]

File: packages/source-storybook/src/index.ts (hunk 1 above)
Comment (tambo) [reply packages/source-storybook/src/index.ts:27]: Agreed. It's probably going to be rarely needed.

Comment (mark-tate) [reply packages/source-storybook/src/index.ts:27]: done

**Pull request merged**